result-reactor/src/main/kotlin/com/virtuslab/basetypes/result/reactor/FluxEither.kt

```diff
@@ -1,6 +1,73 @@
 package com.virtuslab.basetypes.result.reactor
 
 import arrow.core.Either
+import arrow.core.flatMap
+import arrow.core.left
 import arrow.fx.reactor.FluxK
+import arrow.fx.reactor.handleErrorWith
+import arrow.fx.reactor.k
+import reactor.core.publisher.Flux
 
-typealias FluxEither<E, V> = FluxK<Either<E, V>>
\ No newline at end of file
+typealias FluxEither<E, V> = FluxK<Either<E, V>>
+
+fun <S, E, S2> FluxEither<E, S>.mapRight(mapper: (S) -> S2): FluxEither<E, S2> =
+    this.map {
+        it.map(mapper)
+    }
+
+fun <S, E, E2> FluxEither<E, S>.mapLeft(mapper: (E) -> E2): FluxEither<E2, S> =
+    this.map {
+        it.mapLeft(mapper)
+    }
+
+// TODO refine name
+fun <S, E, S2> FluxEither<E, S>.flatMapEither(mapper: (S) -> Either<E, S2>): FluxEither<E, S2> =
+    this.map {
+        it.flatMap(mapper)
+    }
+
+fun <S, E, S2> FluxEither<E, S>.flatMapRight(mapper: (S) -> FluxEither<E, S2>): FluxEither<E, S2> =
```

Comment:
If its FluxK isn't there some import of flatMap that works this way?

Reply:
there is FluxK<T>.flatMap(f: (T)->FluxK)